### PR TITLE
Ocwa import tools

### PR DESCRIPTION
**Pull request by ateucher**
Tools (one exported function: `ocwa_branch_export()`) to prepare the repo for import into OCWA by:

1. Creating a new branch
2. Removing files that can't be imported - these are listed in the `_ocwaignore` file in the root of the repo
3. Cleaning `README.md` to comment out references to images and links that won't be available in the SRE. Basically at this point the sticker and the badges.
4. Committing the changes from 2 and 3 to the new branch and pushing that to GitHub. This branch can then be used as the basis for an import into OCWA.

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -48,4 +48,107 @@ dip.file.exists <- function(x) {
...
+#' so that it is not using images or URLs that can't be accessed in the SRE.
+#'
+#' Then checks out a branch in `{dipr}` and commits those changes to that branch.
+#' Then use that branch to import into OCWA.
+#'
+#' @param branch branch name to prepare for OCWA. Default `"ocwa-import"`
+#'
+#' @return hash of commit
+#' @export
+ocwa_branch_export <- function(branch = "ocwa-import") {
```
Comment (boshek) [inline R/utils.R:110]: I wonder if this would benefit from an `ask` parameter _or_ an ask option.

File: R/utils.R [hunk 2]
```diff
@@ -48,4 +48,107 @@ dip.file.exists <- function(x) {
...
+      "The gert package is required for this function"
+    )
+  }
+
+  curr_branch <- gert::git_branch()
+
+  proceed <- utils::askYesNo(paste0("This will prepare a branch named ", branch,
+                      "\nwith the current changes in ", curr_branch,
+                      ".\nWould you like to proceed?\n"))
+  stopifnot(isTRUE(proceed))
```
Comment (boshek) [inline R/utils.R:122]: Maybe a more informative message?
```suggestion
  if(!isTRUE(proceed)) stop("export branch not created", call. = FALSE)
```

File: .Rbuildignore [hunk 3]
```diff
@@ -20,6 +20,7 @@ data-raw/
 ^docs$
 ^pkgdown$
 
+^_ocwaignore$
```
Comment (boshek) [inline .Rbuildignore:23]: Is there a reason for the `_` instead of `.`?

**Review by boshek — APPROVED** [review]
This is amazing. Well done. I think adding a NEWS item and also your name into the DESCRIPTION file would be worthwhile in this PR.

Just a couple questions.

File: .Rbuildignore (hunk 3 above)
Comment (ateucher) [reply .Rbuildignore:23]: The RStudio file browser only whitelists certain dotfiles for display - e.g., `.gitignore`, `.Rbuildignore` etc. - so you can create a `.ocwaignore` file, but then you can't see it

File: R/utils.R (hunk 1 above)
Comment (ateucher) [reply R/utils.R:110]: Do you mean so you have the option to skip the interactive askYesNo, or should I add more asks along the way? It does do quite a lot with no intervention!

Comment (ateucher) [reply R/utils.R:110]: Do you mean so you have the option to skip the interactive askYesNo, or should I add more asks along the way? It does do quite a lot with no intervention!

**Comment by ateucher** [comment]
@boshek for NEWS and DESCRIPTION - what version should I set? A dev version or 1.1 or???

**Comment by boshek** [comment]
I think leave it was a dev version? My usual process for versioning is that when the inside/outside repos are in the same state, you version it. Open to other versioning schemes though.

**Review by boshek — APPROVED** [review]

**Pull request merged**